README.md

```diff
@@ -1,13 +1,119 @@
-# 🥐 Croissant
+## Summary
 
-[![CI](https://github.com/mlcommons/croissant/actions/workflows/ci.yml/badge.svg)](https://github.com/mlcommons/croissant/actions/workflows/ci.yml/badge.svg)
-[![Python 3.10+](https://img.shields.io/badge/python-3.10+-blue.svg)](https://www.python.org/downloads/)
+Croissant 🥐 is a high-level format for machine learning datasets that combines metadata, resource file descriptions, data structure, and default ML semantics into a single file; it works with existing datasets to make them easier to find, use, and support with tools.
 
-## Task force
+Croissant builds on [schema.org](https://schema.org/), and its Dataset vocabulary, a widely used format to represent datasets on the Web, and make them searchable.
 
-All information at [mlcommons.org/croissant](mlcommons.org/croissant).
+Croissant is currently under development by the community, with the official launch planned at the ICML DMLR workshop on July 29th, 2023. 
 
-## 🥐 ML Croissant
+## Why a standard format for ML datasets?
 
-ML Croissant is a Python library that verifies and loads Croissant datasets.
-Please, refer to the corresponding [README.md](./python/format).
+Datasets are the source code of machine learning (ML), but working with ML datasets is needlessly hard because each dataset has a unique file organization and method for translating file contents into data structures and thus requires a novel approach to using the data. We need a standard dataset format to make it easier to find and use ML datasets and especially to develop tools for creating, understanding, and improving ML datasets.
+
+## The Croissant Format
+
+Croissant 🥐 is a high-level format for machine learning datasets. Croissant brings together four rich layers (in a tasty manner, we hope 😉):
+* Metadata: description of the dataset, including responsible ML aspects
+* Resources: one or more files or other sources containing the raw data
+* Structure: how the raw data is combined and arranged into data structures for use
+* ML semantics: how the data is most often used in an ML context
+
+## Simple Example
+
+Here is an extremely simple example of the croissant format, with comments showing the four layers:
+
+```
```

Comment:
You can replace code block with: https://github.com/mlcommons/croissant/blob/e9bbcbd903372339a951c328e8172b7b82582684/datasets/recipes/minimal_recommended.json#L1-L50

Which will render the code properly., similarly to https://github.com/mlcommons/croissant/blob/main/docs/howto/specify-splits.md.